numbers_and_words.gemspec

```diff
@@ -218,13 +223,21 @@ Gem::Specification.new do |s|
   s.homepage = "http://github.com/kslazarev/numbers_and_words".freeze
   s.licenses = ["MIT".freeze]
   s.required_ruby_version = Gem::Requirement.new(">= 2.7.0".freeze)
-  s.rubygems_version = "3.4.8".freeze
+  s.rubygems_version = "3.3.7".freeze
   s.summary = "Spell out numbers in several languages".freeze
 
-  s.specification_version = 4
+  if s.respond_to? :specification_version then
+    s.specification_version = 4
+  end
 
-  s.add_runtime_dependency(%q<i18n>.freeze, ["<= 2"])
-  s.add_development_dependency(%q<jeweler>.freeze, ["~> 2"])
-  s.add_development_dependency(%q<rake>.freeze, ["~> 12"])
+  if s.respond_to? :add_runtime_dependency then
+    s.add_runtime_dependency(%q<i18n>.freeze, ["<= 2"])
+    s.add_development_dependency(%q<jeweler>.freeze, ["~> 2"])
+    s.add_development_dependency(%q<rake>.freeze, ["~> 12"])
+  else
+    s.add_dependency(%q<i18n>.freeze, ["<= 2"])
+    s.add_dependency(%q<jeweler>.freeze, ["~> 2"])
+    s.add_dependency(%q<rake>.freeze, ["~> 12"])
+  end
```

Comment:
This seems unrelated, you should revert these parts. If there's a reason to make this dynamic, let's put this in another PR.

Reply:
done